packages/types/lib/ExpressionEvaluator.ts

```diff
@@ -0,0 +1,73 @@
+import type * as RDF from '@rdfjs/types';
+import type { Algebra as Alg } from 'sparqlalgebrajs';
+import type { IActionContext } from './IActionContext';
+
+/**
+ * Interface that performs a specific aggregation.
+ * You can put bindings and when all bindings have been put, request the result.
+ */
+export interface IBindingAggregator {
```

Comment:
```suggestion
export interface IBindingsAggregator {
```
(to be aligned with the bus name)